fenix/app/src/main/res/values/strings.xml

```diff
@@ -2235,9 +2235,13 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check" moz:removedIn="121" tools:ignore="UnusedResources">Open Review quality check</string>
```

Comment:
Since this string only landed in Nightly and it's unused, I assume you could just remove it?